.circleci/config.yml

```diff
@@ -8,53 +11,53 @@ executors:
         environment:
           CXX: g++-8
           CC: gcc-8
-          BUILD_FLAGS: -j
-          CTEST_FLAGS: -j4 --output-on-failure
+          BUILD_FLAGS: 
+          CTEST_FLAGS: --output-on-failure
```

Comment:
I'm fine with this as we'd rather have reliable builds than fast ones; but when you looked, did you happen to notice if the memory issues happened all in build or all in test? We'd obviously rather have both reliable and fast wherever we can :)

Reply:
I have seen both. I have seen memory failures with perfcheck and I have seen compilers just crashing randomly.